### DISCUSSION TITLE
The ugly cutter2.0

### DISCUSSION THREAD

**Discussion by sonwnja** [rizinorg/cutter#2677] [General]
It has become weird since 2.0, a dos window appears, which is simply ugly

**Comment by xarkes** [comment]
Hello, I think you are referring to this known issue: https://github.com/rizinorg/cutter/issues/2657
I believe no technical solution was decided for now, but if you are aware of how to fix that, don't hesitate to comment the issue with a suggestion.

**Comment by sonwnja** [comment]
Why has it changed from 1.12 to the current situation? What happened during this period? I have always been a loyal fan, but this situation I think is very bad




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "rizinorg/cutter"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月22日(星期四) 下午5:50
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [rizinorg/cutter] The ugly cutter2.0 (#2677)





 
Hello, I think you are referring to this known issue: #2657
 I believe no technical solution was decided for now, but if you are aware of how to fix that, don't hesitate to comment the issue with a suggestion.

**Reply by karliss** [reply]
> Why has it changed from 1.12 to the current situation? What happened during this period?

Some of the functionality in Windows version of Cutter <= 1.12  that worked on other operating system was simply not working and messages were getting lost. It was chosen that having the window displayed in a tool which is target at technical audience is less bad than output getting lost.

**Comment by p1yang** [comment]
How do I know how many functions are analyzed in my program？

**Reply by ret2libc** [reply]
Hi @p1yang ! I think you can create a separate discussion for completely different questions than what the original topic was about.

Anyway, you can see the number of functions in the Dashboard view:
![image](https://user-images.githubusercontent.com/562321/155118878-3f790efc-3224-4850-8359-4d6c7ed3b4fd.png)

**Reply by p1yang** [reply]
thanks,I find it
